clients/fides-js/src/lib/ui-utils.ts

```diff
@@ -0,0 +1,6 @@
+export const blockPageScrolling = () => {
+  document.body.style.overflowY = "hidden";
+};
+export const unblockPageScrolling = () => {
+  document.body.style.overflowY = "auto";
```

Comment:
Do we know that setting "auto" will for sure revert functionality of all kinds of sites back to the original? Or should we be tracking the original `document.body.style.overflowY` just in case that was set to something other than "auto" before we touched it? I'm unsure here, but lemme know your thoughts!

Reply:
I will try adding a class with styles. That way it 100% reverts to original